models/org.go

```diff
@@ -285,32 +301,29 @@ type OrgUser struct {
 }
 
 // IsOrganizationOwner returns true if given user is in the owner team.
-func IsOrganizationOwner(orgID, uid int64) bool {
-	has, _ := x.
+func IsOrganizationOwner(orgID, uid int64) (bool, error) {
+	return x.
 		Where("is_owner=?", true).
 		And("uid=?", uid).
 		And("org_id=?", orgID).
 		Get(new(OrgUser))
```

Comment:
`Get` -> `Exist` will has better performance.
`Exist(new(OrgUser))` -> `Table("org_user").Exist()` will has better performance and less memory usage.

Reply:
done